permits/tests/test_a_permit_request.py

```diff
@@ -2799,6 +2799,48 @@ def test_email_to_author_is_sent_when_secretariat_acknowledges_reception(self):
             mail.outbox[0].message().as_string(),
         )
 
+    def test_email_to_author_is_sent_when_secretariat_checks_notify(self):
+        user = factories.UserFactory(email="user@geocity.com")
+        permit_request = factories.PermitRequestFactory(
+            status=models.PermitRequest.STATUS_SUBMITTED_FOR_VALIDATION,
+            administrative_entity=self.administrative_entity,
+            author=user.permitauthor,
+        )
+        factories.PermitRequestGeoTimeFactory(permit_request=permit_request)
+        response = self.client.post(
+            reverse(
+                "permits:permit_request_detail",
+                kwargs={"permit_request_id": permit_request.pk},
+            ),
+            data={
+                "status": models.PermitRequest.STATUS_AWAITING_SUPPLEMENT,
+                "notify_author": "1",
+                "reason": "Testing email sending",
+                "action": models.ACTION_AMEND,
+            },
+            follow=True,
+        )
+
+        permit_request.refresh_from_db()
+        self.assertEqual(
+            permit_request.status, models.PermitRequest.STATUS_AWAITING_SUPPLEMENT
+        )
+        self.assertContains(response, "compléments")
+        self.assertEqual(len(mail.outbox), 1)
+        self.assertEqual(mail.outbox[0].to, ["user@geocity.com"])
+        self.assertEqual(
+            mail.outbox[0].subject, f"Votre demande #{permit_request.pk} à changé"
```

Comment:
Maybe we should refer to the request WOT name(s) instead of the requests' primary key in the email's body, it seems like it would be more relatable for the user.